typedoc-theme/src/lib/theme.ts

```diff
@@ -54,14 +54,14 @@ export default class NxMarkdownTheme extends MarkdownTheme {
         directory: '.',
         template: this.getReflectionTemplate(),
       },
+      {
+        kind: [ReflectionKind.TypeAlias],
+        isLeaf: true,
+        directory: '.',
+        template: this.getReflectionMemberTemplate(),
+      },
```

Comment:
Why was this moved?  I don't think it makes a difference in the output.

Reply:
Huh, I would have sworn it did, but removing it and running "nx documentation" resulted in no changes. I'll revert it.